crates/db_schema/src/impls/post.rs

```diff
@@ -54,6 +54,19 @@ impl Crud for Post {
       .await
   }
 
+  /// Ignores unpublished scheduled posts
+  /// TODO: with this filter its not possible to view your own scheduled posts.
+  ///       need to find another way.
+  async fn read(pool: &mut DbPool<'_>, id: Self::IdType) -> Result<Option<Self>, Error> {
+    let conn = &mut *get_conn(pool).await?;
+    post::table
+      .find(id)
+      //.filter(post::scheduled_publish_time.is_null())
```

Comment:
Can delete I think.

Reply:
Done, it seems that this check is not needed as the method is only used internally